src/DatePicker/CalendarMonth.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
```

Comment:
That's a dev dependency. That won't work once released.
The source code is here https://github.com/facebook/react/blob/master/src/test/ReactTestUtils.js#L93. Couldn't we inline it?